modmail/utils/pagination.py

```diff
@@ -212,15 +243,6 @@ async def interaction_check(self, interaction: Interaction) -> bool:
         )
         return False
 
-    def get_footer(self) -> str:
-        """Returns the footer text."""
-        self.embed.description = self._pages[self.index]
-        page_indicator = f"Page {self.index+1}/{len(self._pages)}"
-        footer_txt = (
-            f"{self.footer_text} ({page_indicator})" if self.footer_text is not None else page_indicator
-        )
-        return footer_txt
-
     def update_states(self) -> None:
```

Comment:
Bikeshed: name this `update` to make more sense?

Reply:
update states does not actually send the message, that is left to send_page